Tests/Common/LoRaEnumerable.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tests.Common
+{
+    using System.Collections.Generic;
+
+    internal static class LoRaEnumerable
+    {
+        public static IEnumerable<T> Repeat<T>(T value)
```

Comment:
I would suggest to make this more obvious that it is repeating over and over again without ending. RepeatAlways?

Reply:
3d6d917fc233e5b26070b75cce4521611ba418c5